ui/admin/app/templates/application.hbs

```diff
@@ -89,14 +89,28 @@
                     {{#each this.featureEdition.editions as |edition|}}
                       <dropdown.item>
                         <radioGroup.radio
-                          @label={{t (concat 'editions.' edition)}}
+                          @label={{edition}}
                           @value={{edition}}
                         />
                       </dropdown.item>
                     {{/each}}
                   </form.radioGroup>
                 </Rose::Form>
               </dropdown.section>
+              <dropdown.separator />
```

Comment:
The current dropdown is `Rose`, Can we use an HDS dropdwon here?

Reply:
Currently, HDS doesn't support radio button within a dropdown

Reply:
Not really, because this is in main app header, which doesn't support HDS components.  The overall app chrome can be migrated in the future when HDS supports all of the components that appear within it.